packages/devtools_app/lib/src/screens/memory/memory_heap_tree_view.dart

```diff
@@ -1464,11 +1466,12 @@ class MemoryHeapTableState extends State<MemoryHeapTable>
         selectionNotifier: controller.selectionSnapshotNotifier,
       );
 
-      return controller.groupByTreeTable;
+      result = controller.groupByTreeTable;
     } else {
       // Nothing collected yet (snapshots/analyses) - return an empty area.
-      return const SizedBox();
+      result = const SizedBox();
     }
+    return OutlineDecoration(child: result);
```

Comment:
does an OutlineDecoration around an empty SizedBox really do what we think it does? since the SizedBox has size 0,0 I would expect to see nothing.

Reply:
Ah yeah, good point. Doesn't seem to make any difference.